controllers/articles.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { fetchArticleById } = require('../models/articles.models');
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: 'Invalid article ID' });
+  }
```

Comment:
You don't need to manually check the article_id data type - psql will do that for you. Let this run through without this block of code and catch the error that psql gives you by trying to use an incorrect data type for the id.